### ISSUE TITLE
Screen reader announces incorrect number of table rows (Chrome, NVDA)

### ISSUE THREAD

**Issue by pbeleckis** [ag-grid/ag-grid#4024]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Providing a Reproducible Scenario**
Default plunker from ag-grid documentation:
https://www.ag-grid.com/javascript-grid-selection/#example-selection-checkbox-filtered

- Click on **Filter Only Swimming** button to filter some data
- Focus first table header cell (**Athlete**)
- NVDA screen reader announces - "table with **35** rows and 8 columns Athlete clickable"
- Now scroll the table to the bottom
- Focus first table header cell (**Athlete**) again
- NVDA screen reader announces - "table with **48** rows and 8 columns Athlete clickable"

Notes:
- This issue is reproducible even if we do not filter table data (_by omitting the first step_)
   For example [another plunker](https://www.ag-grid.com/javascript-grid-selection/#example-group-selection) can be use
- Other browsers seem to work correctly

**Expected behavior**
The screen reader should announce the correct number of rows.

**Please tell us about your environment:**
Windows 10

* **ag-Grid version:** 23.2.0

* **Browser:**
Google Chrome Version 84.0.4147.125

* **Screen reader:**
NVDA version 2020.2

* **Language:** React 16.13.1

![image](https://user-images.githubusercontent.com/50491863/90386524-5d62fb80-e08d-11ea-8ad0-cae7436470ca.png)

**Comment by LouisMoore-agGrid** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by kiril-matev** [comment]
Hi team,

Thank you for reporting this.

We've been able to reproduce this issue.

We've investigated this and found that the value of the ARIA label in ag-grid that stores the row count is correct and doesn't change while the user is scrolling. 

We've determined this is an issue with NVDA which is announcing an incorrect value for aria-rowcount after scrolling. This is actually visible in your screenshot as well which is showing that aria-rowcount=35, but NVDA makes a second announcement with a different number for rowcount. This is the bug that needs to be resolved in NVDA - why is it announcing a different number, and where is it getting that number from?

We have added an issue to our backlog and we are tracking it with the following reference and description:
**AG-4520 [Accessibility] Announce correct row count in grid**

As we've confirmed that the aria-rowcount value is correct and doesn't change, this is an NVDA issue which announces a larger rowcount later on even though the . We will close this issue on our side and enter additional information on that issue with the link to the question on the NVDA forum about this when we enter one.

In the meantime, I would suggest you also submit this as a question on the NVDA forums to investigate this issue as the aria-rowcount value clearly doesn't change in ag-Grid, but NVDA announces a different number.

Thanks again for bringing this up with us.
